README.md

```diff
@@ -160,3 +160,15 @@ These images do not expire, and you can keep them in your repository for as long
 # Contributing
 
 This is an open source project and contributions are welcome.  Please see the [contributing guidelines](./CONTRIBUTING.md) to get started.
+
+# Versioning
```

Comment:
# Versioning of the Pega Helm repository

Pega may release new versions of this Pega Helm chart released at any time. (Reviewers: do you want to add something about givving clients a heads up about future versions or is it not required?) Versions are defined as major.minor.patch, as follows:

1. Major: Used when required values.yaml or the config maps (config/deploy) do not adequately support generating Kubernetes yaml and deploy using previously released Docker image versions.
  - Example: A dependency on a specific Pega Platform version, or a dependency on either a required pega-web-ready or a pega-installer-ready image change.
2. Minor: Used when the older values.yaml or the config maps (config/deploy) no longer support the Helm parameters to adopt new functionality and to generate the Kubernetes yaml. Minor may also be used for new features.
  - Example: A dependency on an optional pega-web-ready change or pega-installer-ready image change.
3. Patch: Any other change, such opt-in enhancements  and bugs fixes, that Pega deems important enough to address between minor releases.
  - Example: Introducing a new, optional parameter that defaults to a previously hard-coded value.

Reply:
After more discussion with John and others, we've decided to take on a more standard versioning approach (by just using semantic versioning exactly as it is).